public/locales/tr/send.ftl

```diff
@@ -15,6 +16,7 @@ encryptingFile = Şifreleniyor...
 decryptingFile = Şifre çözülüyor...
```

Comment:
@flodo Yeah, I think the thing that caught my eye is that some locales were using both.

If we just look at public/locales/en-us/send.ftl, we should have four instances of [`…`](https://github.com/mozilla/send/blob/41f68e27350395a80aae6aeefa1cf5c57612f496/public/locales/en-US/send.ftl#L15-L18) in the send.ftl file:

```sh
$ git grep "…" public/locales/en-US/send.ftl | wc -l # 4
```

So any translations above that have all four instances using `...` would be fine (dsb, es-ES, es-MX, hsb, ja, kab).

That leaves us with the following:

```sh
$ git grep -A 3 "importingFile = " public/locales/(cy|de|dsb|es-ES|es-MX|hsb|ja|kab|ms|nb-NO|nn-NO|pt-BR|pt-PT|ru|sk|sl)/send.ftl

public/locales/cy/send.ftl:importingFile = Mewnforio…
public/locales/cy/send.ftl-verifyingFile = Wrthi'n gwirio
public/locales/cy/send.ftl-encryptingFile = Wrthi'n amgryptio...
public/locales/cy/send.ftl-decryptingFile = Wrthi'n dadgryptio
--
public/locales/de/send.ftl:importingFile = Wird importiert…
public/locales/de/send.ftl-verifyingFile = Wird überprüft…
public/locales/de/send.ftl-encryptingFile = Wird verschlüsselt
public/locales/de/send.ftl-decryptingFile = Wird entschlüsselt...
--
public/locales/ms/send.ftl:importingFile = Mengimport…
public/locales/ms/send.ftl-verifyingFile = Mengesahkan...
public/locales/ms/send.ftl-encryptingFile = Mengenkripsi...
public/locales/ms/send.ftl-decryptingFile = Mengenkripsi...
--
public/locales/nb-NO/send.ftl:importingFile = Importerer…
public/locales/nb-NO/send.ftl-verifyingFile = Verifiserer...
public/locales/nb-NO/send.ftl-encryptingFile = Krypterer...
public/locales/nb-NO/send.ftl-decryptingFile = Dekrypterer...
--
public/locales/nn-NO/send.ftl:importingFile = Importerer…
public/locales/nn-NO/send.ftl-verifyingFile = Stadfestar…
public/locales/nn-NO/send.ftl-encryptingFile = Krypterer…
public/locales/nn-NO/send.ftl-decryptingFile = Dekrypterer...
--
public/locales/pt-BR/send.ftl:importingFile = Importando…
public/locales/pt-BR/send.ftl-verifyingFile = Verificando…
public/locales/pt-BR/send.ftl-encryptingFile = Criptografando...
public/locales/pt-BR/send.ftl-decryptingFile = Descriptografando...
--
public/locales/sk/send.ftl:importingFile = Importuje sa…
public/locales/sk/send.ftl-verifyingFile = Overuje sa…
public/locales/sk/send.ftl-encryptingFile = Šifruje sa...
public/locales/sk/send.ftl-decryptingFile = Dešifruje sa...
--
public/locales/sl/send.ftl:importingFile = Uvažanje …
public/locales/sl/send.ftl-verifyingFile = Preverjanje …
public/locales/sl/send.ftl-encryptingFile = Šifriranje ...
public/locales/sl/send.ftl-decryptingFile = Dešifriranje ...
```